packages/dotcom-ui-header/package.json

```diff
@@ -29,15 +29,14 @@
     "@financial-times/logo-images": "^1.10.1",
     "@svgr/core": "^5.0.0",
     "camelcase": "^6.0.0",
-    "check-engine": "^1.10.1",
-    "@financial-times/o-header": "^13.0.0"
+    "check-engine": "^1.10.1"
   },
   "peerDependencies": {
-    "@financial-times/o-header": "^13.0.0",
     "@financial-times/logo-images": "^1.10.1",
+    "@financial-times/o-header": "13.0.1",
```

Comment:
o-header is a peer dependency now. Is this intended?

Reply:
Oh, it was removed from `devDependencies` and it's referenced in two places 🤔

Reply:
it only needs to be peer dependency